src/App.js

```diff
@@ -1,12 +1,81 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import Pokemon from "./Pokemon";
+import axios from 'axios';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokenames, setPokenames] = useState(null);
```

Comment:
Als dit altijd een array is, dan zou ik dit als default value een lege array laten zijn i.p.v. null